tldw_Server_API/app/api/v1/endpoints/media/transcription_models.py

```
@@ -0,0 +1,28 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List
+
+from fastapi import APIRouter, status
+
+from tldw_Server_API.app.core.Ingestion_Media_Processing.transcription_models import (  # type: ignore  # noqa: E501
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)